src/main/java/com/gepardec/mega/zep/rest/mapper/RegularWorkingHoursMapMapper.java

```diff
@@ -0,0 +1,36 @@
+package com.gepardec.mega.zep.rest.mapper;
+
+import com.gepardec.mega.zep.ZepServiceException;
+import com.gepardec.mega.zep.rest.dto.ZepRegularWorkingTimes;
+import jakarta.enterprise.context.ApplicationScoped;
+
+import java.time.DayOfWeek;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+
+@ApplicationScoped
+public class RegularWorkingHoursMapMapper implements Mapper<Map<DayOfWeek, Duration>, ZepRegularWorkingTimes> {
+
+    @Override
+    public Map<DayOfWeek, Duration> map(ZepRegularWorkingTimes zepRegularWorkingTimes) {
+
+        try{
+
+            Map<DayOfWeek, Duration> regularWorkingHours = new HashMap<>();
```

Comment:
@sgartner03 wenn alle keys in der map enums vom selben typ sind ist new EnumMap<>() besser als eine HashMap